src/osip-simulation-view/src/main/java/edu/kit/pse/osip/simulation/view/dialogs/AboutDialog.java

```diff
@@ -1,7 +1,137 @@
 package edu.kit.pse.osip.simulation.view.dialogs;
 
+import javafx.scene.layout.GridPane;
+import javafx.scene.image.Image;
+import javafx.scene.text.Font;
+import javafx.scene.text.Text;
+import javafx.geometry.Insets;
+import javafx.scene.Scene;
+import javafx.scene.image.ImageView;
+
+import java.io.IOException;
+import java.io.InputStreamReader;
+
+import edu.kit.pse.osip.core.utils.language.Translator;
+import edu.kit.pse.osip.simulation.view.main.ViewConstants;
+import javafx.scene.control.ScrollPane;
+import javafx.scene.layout.Priority;
+import javafx.scene.layout.HBox;
+import javafx.scene.text.TextFlow;
+
 /**
  * This window shows information about the creators of OSIP.
+ * @author Maximilian Schwarzmann
+ * @version 1.0
  */
-public class AboutDialog extends javafx.stage.Stage {
+public class AboutDialog extends javafx.stage.Stage {    
+    private static final int FONT_SIZE = ViewConstants.FONT_SIZE;
+    private static final int MIN_WINDOW_WIDTH = 800;
+    private static final int MIN_WINDOW_HEIGHT = 500;
+    private static final String PROG_NAME = "OSIP";
+    
+    /**
+     * Constructor of AboutDialog
+     */
+    public AboutDialog() {
+        configure();
+    }
+
+    private void setDialogSize() {
+        this.setMinWidth(MIN_WINDOW_WIDTH);
+        this.setMinHeight(MIN_WINDOW_HEIGHT);
+    }
+
+    private void configureIcon(Image image, ImageView icon) {
+        icon.setImage(image);
+        icon.setFitWidth(8 * FONT_SIZE);
+        icon.setPreserveRatio(true);
+        icon.setSmooth(true);
+    }
+    
+    private void configureScrollPane(ScrollPane scrollPane, TextFlow textFlow) {
+        scrollPane.setFitToWidth(true);
+        scrollPane.setContent(textFlow);
+        scrollPane.setPadding(new Insets(ViewConstants.ELEMENTS_GAP));
+    }
+
+    private Text getText(String id, Font font) {
+        Text text = new Text();
+        text.setText(Translator.getInstance().getString(id) + "\n");
+        if (font != null) {
+            text.setFont(font);
+        }
+        return text;
+    }
+    
+    private Text newLine() {
+        return new Text("\n");
+    }
+    
+    private Text getLicenseText(String path) {
+        InputStreamReader reader = null;
```

Comment:
Meine Änderungsvorschläge beziehen sich auch auf den Über-Dialog in der Überwachungskonsole.